packages/cli-kit/src/public/node/custom-oclif-loader.ts

```diff
@@ -1,18 +1,22 @@
-import {cwd, joinPath} from './path.js'
 import {fileExistsSync} from './fs.js'
+import {cwd, joinPath} from './path.js'
 import {Command, Config} from '@oclif/core'
 import {Options} from '@oclif/core/lib/interfaces/plugin.js'
 
 export class ShopifyConfig extends Config {
   constructor(options: Options) {
     const path = sniffForPath() ?? cwd()
     if (fileExistsSync(joinPath(`${path}`, 'package.json'))) {
+      // Hydrogen is bundled, but we still want to support loading it as an external plugin for two reasons:
```

Comment:
In the experimental version you linked, I'm getting warnings when running with `DEBUG="*"` to tell me about OCLIF's loading process. Not sure if due to experiment or something not right here.

(there's also a few like this `  config:ts-path Skipping typescript path lookup for /Users/shauns/Library/pnpm/global/5/.pnpm/@shopify+cli@0.0.0-experimental-20240530091455/node_modules/@shopify/cli because NODE_ENV is NOT "test" or "development" +17ms` , which maybe says something is misconfigured, as we wouldn't expect to even try to load as TS, and 17ms is a bit of a high tax to pay)

Reply:
Lookint at oclif/core those `Skipping typescript path lookup` are expected, it's printed when `isProduction` is true. Not sure why it takes that much time though.

There are a few warnings because the manifest is not updated to include the experimental version number, but that's ok, is an experimental thing only. 

I don't see any other relevant warnings, you do see anything else?